app/controllers/api/v1/upload_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+module Api
+  module V1
+    class UploadController < ApplicationController
+      def create
+        upload = Upload.new(
```

Comment:
You can change this object into a [Form object](https://nimblehq.co/compass/development/code-conventions/ruby/ruby-on-rails/#forms)